### ISSUE TITLE
Refactor Locales to trust the Path

### ISSUE THREAD

**Issue by MaximumCrash** [makerdao/community-portal#76]
Currently a number of our locale dependent components (Breadcrumbs, Sidenav, ect.) have been the victim of pure component statefulness. In the PageContext we set the default locale to en for the state of the app. The problem with this is that every time you go to a new page in a different locale it goes "EN" -> "EN" -> "FR". This is not favorable, and is a suspect of me overengineering the locale system. 

I've opted for a better philosophy for moving forward for better stability. **Trust the path** 

Our URL path is our source of truth, if a page doesn't exist it's because **it doesn't exist**. While we can do our best with locale fallbacks to "en" there will be cases where data in our fallback doesn't exist either. **This is the worst case scenario. If you've done this, you've gone off the beaten path for this application.** 

TLDR Solution:
Remove the need for saving locale in localStorage and for the app. Use the route and split it to grab the locale. 
The index page is fine for grabbing initial locale since it acts primarily as a router to the correct locales index page.

**Comment by MaximumCrash** [comment]
Closed byu #85
